releasenotes/notes/fix-trace-agent-goroutine-leak-c3be040113bbd6d2.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix goroutine leak in prevent 100% cpu usage due to infinite spin loop when
```

Comment:
Can you please prefix the release note with `APM: `? It helps us differentiate between trace agent changes and other changes in the final change log.